Source/WebCore/animation/ScrollTimeline.h

```diff
@@ -43,8 +44,6 @@ class ScrollableArea;
 
 struct TimelineRange;
 
-enum class Scroller : uint8_t { Nearest, Root, Self };
-
```

Comment:
Why is that change necessary?

Reply:
Moved build fixes to https://github.com/WebKit/WebKit/pull/39640.